data/scenarios/Testing/397-wrong-missing.yaml

```diff
@@ -0,0 +1,42 @@
+version: 1
+name: Test issue 397 (wrong device reported missing)
+description: |
+  A scenario designed to trigger issue 397, where in certain
+  situations involving a capability with no candidate devices,
+  an incorrect error message was generated reporting some other
+  device to be missing.
+  https://github.com/swarm-game/swarm/issues/397
+objectives:
+  - condition: |
+      as base { ishere "rock" }
+    goal:
+    - |
+      This is a dummy condition that just ensures the base has
+      finished trying to run the problematic `build` command.  The
+      scenario *should* generate an error message; what really care
+      about is whether the generated error message is correct, which
+      is checked in test/integration/Main.hs .
+solution: |
+  build {require 1 "rock"; wait 4; place "rock"};
```

Comment:
I guess you could just check `t <- time; return (t==5)` if that is the intent. :slightly_smiling_face: